### PR TITLE
Reporting modal

### LINKED ISSUE #2910: Disputing/Reporting form header
Modal view:
https://www.figma.com/file/aAzKHh4cA6OT2t7WFv2BQ7fB/Reporting-and-Disputing?node-id=794%3A347

 * initial reporting the header shows `report on this market`
 * after market has a tentative winning outcome the header states `dispute or support this market's tentative winning outcome`

 * Market information is the same for all market states in the form.
https://www.figma.com/file/aAzKHh4cA6OT2t7WFv2BQ7fB/Reporting-and-Disputing?node-id=231%3A69030

Screen Shot 2019-08-28 at 12 45 25 PM

mobile view:

https://www.figma.com/file/aAzKHh4cA6OT2t7WFv2BQ7fB/Reporting-and-Disputing?node-id=0%3A1

### LINKED ISSUE #2911: Reporting/Disputing form REP balance bar
https://www.figma.com/file/aAzKHh4cA6OT2t7WFv2BQ7fB/Reporting-and-Disputing?node-id=231%3A69030

 * the REP balance section isn't shown when market is in Open Reporting

stub out buttons for `Purchase REP` and `Convert to REP` still waiting for design.

available REP balances comes from loginAccount REP balance
initial reporter stake comes from -- market info no-show bond amount

mobile:

https://www.figma.com/file/aAzKHh4cA6OT2t7WFv2BQ7fB/Reporting-and-Disputing?node-id=793%3A978 

 * notice on mobile view initial bond amount is hidden

### PR DESCRIPTION
**Pull request by phoebemirman**
closes https://github.com/AugurProject/augur/issues/2910
closes https://github.com/AugurProject/augur/issues/2911

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 💎

 This link will always point to this PR's latest changes: https://peek.run/qo0hj96

File: packages/augur-ui/src/modules/modal/reporting.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      disputeInfo
+    } = market;
+
+    const radioButtons = outcomesFormatted.map(outcome => {
+      let stake = disputeInfo.stakes.find(stake => parseFloat(stake.outcome) === outcome.id);
+      if (!stake) {
+        stake = {
+          tentativeWinning: false,
+          bondSizeCurrent: "0",
+          bondSizeTotal: "1",
```
Comment (bthaile) [inline packages/augur-ui/src/modules/modal/reporting.tsx]: why is `bondSizeTotal` "1"?

**Review by bthaile — APPROVED** [review]
lgtm, one question

File: packages/augur-ui/src/modules/modal/reporting.tsx [hunk 2]
```diff
@@ -79,6 +81,8 @@ export default class ModalReporting extends Component<ModalReportingProps, Modal
       };
     });
 
+    console.log(radioButtons);
```
Comment (bthaile) [inline packages/augur-ui/src/modules/modal/reporting.tsx]: prob don't need console here

**Pull request merged**